files/45cfb579-d44b-4646-ab04-35d5a0eb7c87/text.txt

```diff
@@ -0,0 +1 @@
+some file
```

Comment:
I open PR and it's first what I see. Why haven't you firstly checked PR by yourself? I've already asked it. I won't start reviewing till you attentively double check what code you add.
![image](https://user-images.githubusercontent.com/77664426/116604489-df9f0180-a936-11eb-9262-e3f9180b09c3.png)

Reply:
I can say more. It is responsibility not only PR owner, but also another team member who have approved before providing PR for mentors. In this case it was @Maksim-maevsky

Reply:
Maxim approved before I resolved conflicts between branches. 
Fixed